spring-boot-project/spring-boot-docs/src/docs/asciidoc/features/docker-compose.adoc

```diff
@@ -37,6 +37,7 @@ When this module is included as a dependency Spring Boot will do the following:
 
 NOTE: The `docker compose` or `docker-compose` CLI application needs to be on your path in order for Spring Boot’s support to work correctly.
 
+NOTE: If you wish to run docker compose via `spring-boot-docker-compose` when using `@SpringBootTest` 's tests, the property `spring.docker.compose.skip.in-tests` must be set to `false`
```

Comment:
Can you please replace `` `spring.docker.compose.skip.in-tests` `` with `configprop:spring.docker.compose.skip.in-tests[]`. `configprop` is a macro that runs as part of building the documentation. It verifies that the configuration property exists, helping to ensure the accuracy of the documentation.